Mathlib/Data/Set/Lattice.lean

```diff
@@ -1329,6 +1329,19 @@ noncomputable def unionEqSigmaOfDisjoint {t : α → Set β}
     (⋃ i, t i) ≃ Σi, t i :=
   (Equiv.ofBijective _ <| sigmaToiUnion_bijective t h).symm
 
+@[simp]
+lemma coe_unionEqSigmaOfDisjoint_symm {α β : Type*} {t : α → Set β}
```

Comment:
Naively, I would have expected this named to named `unionEqSigmaOfDisjoint_symm_apply` (and similarly for the other three). Is there some reason against this, @j-loreaux?

(Otherwise, this PR LGTM and I'd maintainer merge it.)